.github/workflows/ci.yml

```diff
@@ -20,23 +20,15 @@ jobs:
     if: "!contains(github.event.head_commit.message, '[skip ci]')"
     runs-on: ubuntu-latest
     steps:
-      - name: 'Checkout'
-        uses: actions/checkout@main
-      - name: 'Terraform Format'
-        uses: hashicorp/terraform-github-actions@master
+      - uses: actions/checkout@main
+      - uses: hashicorp/setup-terraform@v1
         with:
-          tf_actions_version: 0.12.24
-          tf_actions_subcommand: 'fmt'
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          terraform_version: 0.12.24
```

Comment:
Spotted that this doesn't line up with https://github.com/cookpad/terraform-aws-eks/blob/main/.terraform-version
Could we improve this by reading that file to an environment variable to use in `terraform_version` ? Would be nice to only define it in 1 place in the project

Reply:
Yes - we should do something to have them sync.

Not here though - keeping it as 0.12.24 so it's an atomic change.